common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1233,6 +1287,10 @@ def submit_problem(self, data, override_time=False):
         # render problem into HTML
         html = self.get_problem_html(encapsulate=False, submit_notification=True)
 
+        # Withhold success indicator if hiding correctness
+        if not self.correctness_available():
```

Comment:
Can you move this logic up to line 1260 so that all the logic about 'success' is together? Something like:


````
if self.correctness_available():
     success = 'correct'
     for answer_id in correct_map:
         if not correct_map.is_correct(answer_id):
             success = 'incorrect'
else:
    success = 'submitted'